lib/src/phoenix_html_connection.dart

```diff
@@ -1,11 +1,11 @@
-import 'dart:html';
 import 'dart:async';
+import 'dart:html';
 
 import 'package:phoenix_wings/src/phoenix_connection.dart';
 
 /// PhoenixHtmlConnection handles the creation and use
 /// of the underlying websocket connection on browser platforms.
-class PhoenixHtmlConnection extends PhoenixConnection {
```

Comment:
Can you help me understand why this has been renamed? 

It's resulting in this error on CI

```
  error • test/phoenix_socket_html_test.dart:11:46 • Undefined name 'PhoenixHtmlConnection'. Try correcting the name to one that is defined, or defining the
          name. • undefined_identifier
```

Reply:
this is for help keeping the same name: so if your platform is Web or Non-web you'll have the same class name. otherwise you'll have to somehow import the right file. Then no need to import a specific file pending platform.

Reply:
OK, there are parts of the code expecting to consume these classes with the original names that haven't been updated.

Would it be possible to get things working without changing the class names?